[AGM-2542] Add wfpdiag.cab to crush dump manual (#717)
AdguardTeam/KnowledgeBase: docs/adguard-for-windows/solving-problems/dump-file.md

Since dump files can be large, please compress them into an archive before sending them to support. Also, include the AdGuard logs to help us diagnose the issue more effectively.
[Instructions for collecting logs](../adguard-logs).

## Collecting wfpdiag.cab file

> Our support team may request a wfpdiag.cab file (along with a minidump file) to further analyze system logs and diagnose issues. To collect a wfpdiag.cab file please follow the steps:

Reviewer: ```suggestion
Our support team may request a wfpdiag.cab file, along with a minidump file, to analyze system logs more thoroughly and diagnose issues. To collect the file, please follow these steps:
```

1. Enable AdGuard protection.

> 1. Click *Start* and type `cmd` to open Command prompt

Reviewer: ```suggestion
1. Click *Start* and type `cmd` to open Command Prompt
```

> 1. Right-click Command prompt and choose *Run as administrator*

Reviewer: ```suggestion
1. Right-click Command Prompt and choose *Run as administrator*
```

    :::note

    A wfpdiag.cab file is created in your current directory. You can change the location by typing `cd <folder_name>`.

    :::

> 1. Enter the command to start logging: `netsh wfp capture start`

Reviewer: ```suggestion
1. To start logging, enter the following command: `netsh wfp capture start`
```

1. Visit any website to route traffic through AdGuard and log the filtering process

> 1. Enter the command to stop logging: `netsh wfp capture stop`

Reviewer: ```suggestion
1. To stop logging, enter `netsh wfp capture stop`
```
